### DISCUSSION TITLE
How to fully ignore CR/LF?

### DISCUSSION THREAD

**Discussion by mdrmdrmdr** [WinMerge/winmerge#1738] [Q&A] [answered]
How to configure WinMerge, that the following two code blocks, one 3 lines and one 1 line, compare equal? 

```
function(a) {
  return "x";
}
```
and

`function(a) { return "x"; }`

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge does not currently have a setting to consider the codes as identical.

If you use a diff tool that understands syntax, such as the following, it will treat the codes as the same.

https://difftastic.wilfred.me.uk/

**Reply by mdrmdrmdr** [reply]
Too bad, but thanks for the tip :-)
